zipline/assets/__init__.py

```diff
@@ -20,6 +20,15 @@
     make_asset_array,
     CACHE_FILE_TEMPLATE
 )
+from .asset_writer import (
+    NullAssetDBWriterLegacy,
+    AssetDBWriterLegacyFromList,
+    AssetDBWriterLegacyFromDictionary,
+    AssetDBWriterLegacyFromDataFrame,
+    AssetDBWriterLegacyFromReadable,
+    FUTURE_TABLE_FIELDS,
+    EQUITY_TABLE_FIELDS
```

Comment:
Gonna hold off making a call on this one till I've done some initial integration with the AssetMaster. Your suggestion seems sensible though.